### PR TITLE
style:room的樣式

### PR DESCRIPTION
**Pull request by chutienyuan**
image

### CODE REVIEW

File: app/javascript/stylesheets/rooms/room.scss [hunk 1]
```diff
@@ -37,6 +39,13 @@
         @apply border-gray-300;
       }
     }
+    .result-box{
+      width: calc(100% - 65px)!important;
```
Comment (noracami) [inline app/javascript/stylesheets/rooms/room.scss:43]: 😢

**Review by noracami — APPROVED** [review]

File: app/javascript/stylesheets/rooms/room.scss (hunk 1 above)
Comment (RexXKao) [reply app/javascript/stylesheets/rooms/room.scss:43]: 😢

Comment (chutienyuan) [reply app/javascript/stylesheets/rooms/room.scss:43]: 剛剛有把檔案的(width:600px)行內拿掉, 但測試畫面還是出現, 所以還有其他地方有放width:600px?
image

image

Comment (noracami) [reply app/javascript/stylesheets/rooms/room.scss:43]: image

Comment (noracami) [reply app/javascript/stylesheets/rooms/room.scss:43]: https://github.com/we-have-TTA/iCoder/blob/main/app/javascript/controllers/editor_controller.js#L42
https://github.com/we-have-TTA/iCoder/blob/main/app/javascript/controllers/editor_controller.js#L52

**Review by zxcvbnm7548 — APPROVED** [review]

**Review by oSeanChen — APPROVED** [review]

**Review by RexXKao — APPROVED** [review]

**Review by Yojijun — APPROVED** [review]

**Pull request merged**